addons/point_of_sale/models/pos_order.py

```diff
@@ -886,10 +886,13 @@ def set_pack_operation_lot(self, picking=None):
 
     def _prepare_bank_statement_line_payment_values(self, data):
         """Create a new payment for the order"""
+        payment_name = '%s - %s' % (self.name, self.pos_reference)
+        if data.get('payment_name'):
```

Comment:
Here can use ```'payment_name' in data``` ... it's faster than dict.get()
https://repl.it/repls/ModestOvalFields

Reply:
As this has been already accepted in Odoo master, let's not change it for avoiding differences.

Reply:
The problem is that I need to use coercion here in order to avoid an empty string